Apps/Examples/Examples/All Examples/FeatureStateExample.swift

```diff
@@ -0,0 +1,321 @@
+import UIKit
+import MapboxMaps
+
+@objc(FeatureStateExample)
+
+public class FeatureStateExample: UIViewController, ExampleProtocol {
```

Comment:
Is this feature aligned with the android example?

Reply:
Yep, we're aligned with android and gl-js examples!